vehicle/mkvi/software/bms/tasks/openwire.c

```diff
@@ -4,19 +4,115 @@
 #include "vehicle/mkvi/software/bms/bms_config.h"
 #include "vehicle/mkvi/software/bms/can_api.h"
 #include "vehicle/mkvi/software/bms/utils/fault.h"
-
+#include <string.h>
 
 #define NO_OPEN_WIRES (65535)
+#define NUM_CV_REG    (4)
```

Comment:
doesn't the adbms1818 have 6 cell registers?

6 registers * 3 cells per register = 18 cells per chip